### PR TITLE
fgeyer: Article 6

### PR DESCRIPTION
**Pull request by circumflex3105**

### CODE REVIEW

**Comment by circumflex3105** [comment]
@hendrikebbers ready for review

File: content/posts/2023-06-22-logging-facades-for-java.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+outdated: false
+showInBlog: true
+title: "Logging Facades for Java"
+date: 2023-06-22
+author: hendrik
+excerpt: "Logging is an important part of error analysis. However, consolidating different logging libs in Java applications is always a challenge."
```
Comment (hendrikebbers) [inline content/posts/2023-06-22-logging-facades-for-java.md:7]: adding: If you want to know how several different logging libs in a single application can be handled, this post is for you.

File: content/posts/2023-06-22-logging-facades-for-java.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+categories: [Java]
+origin: https://www.heise.de/blog/Logging-Facades-fuer-Java-7355974.html
+preview_image: "/posts/2023-06-22-logging-facades-for-java/preview.jpg"
+---
+
+After covering best practices and pitfalls in the [first post on Java logging](https://www.heise.de/blog/Best-Practices-und-Anti-Pattern-beim-Logging-in-Java-und-anderen-Sprachen-7336005.html), I now want to look at using logging in a large project.
```
Comment (hendrikebbers) [inline content/posts/2023-06-22-logging-facades-for-java.md:13]: Link to Open elements post

**Comment by circumflex3105** [comment]
@hendrikebbers Review Findings implemented.

**Pull request merged**